spec/statement.dd

```diff
@@ -995,20 +995,42 @@ $(H3 $(LNAME2 foreach_restrictions, Foreach Restrictions))
 
         $(P The aggregate itself must not be resized, reallocated, free'd,
         reassigned or destructed
-        while the foreach is iterating over the elements.
+        while `foreach` is iterating over the elements.
         )
 
 --------------
-int[] a;
-int[] b;
-foreach (int i; a)
+int[] a = [1,2,3];
+
+foreach (int v; a)
 {
-    a = null;       // error
-    a.length += 10; // error
-    a = b;          // error
+    // resizing is unspecified!
```

Comment:
An example of mutation in secret (i.e. inside a function call) would be handy here.